### DISCUSSION TITLE
How to dump Sega Genesis Games

### DISCUSSION THREAD

**Discussion by ccfman2004** [sanni/cartreader#551] [Need help]
How do I dump a Sega Genesis game. 

Hardware v5 r3.

**Comment by sanni** [comment]
https://github.com/sanni/cartreader/wiki/Reading-Mega-Drive-Genesis-games

**Comment by ccfman2004** [comment]
That's what I thought but I can't get my "Not for Retail" version of Sonic the Hedgehog 2 to read. I didn't know if it was one of those weird hard to read games so I bought another Genesis game from a local retro game store which read no issues.

EDIT: I see the 10.2 update clarifies where the Genesis dumping section is. Thank you.
